roles/rng/README.rst

```diff
@@ -1,14 +1,17 @@
-This ansible role install Haveged.
-Haveged is a random number generator.
+This ansible role install rng.
+rngd is a random number generator.
+
+rng-tools include the Daemon to use a Hardware TRNG random number generator.
+The rngd daemon acts as a bridge between a Hardware TRNG.
 
 **Role Variables**
 
 .. zuul:rolevar:: rng_service_name
    :default: haveged
```

Comment:
This should be rngd. While touching this I think it is fine to remove the rng_servie_name parameter and to hardcode rngd inside the task.